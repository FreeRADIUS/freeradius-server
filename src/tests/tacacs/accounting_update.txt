#
#	ARGV: -u tapioca account -f update -c key1=var1
#
