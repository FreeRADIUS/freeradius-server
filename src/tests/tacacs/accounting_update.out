status: SUCCESS
data: b'\x12'
server_msg: b'Accounting-Watchdog Section'
